### PR TITLE
SDK Project base

### PR DESCRIPTION
**Pull request by krishnaKapadia**
This PR creates the base project structure for the SDK, including:
- Rollup: Used to build the project into `umd`, `esm` and `cjs` compatible scripts.
- Prettier: Code formatting
- Jest: Testing framework
- Cross-fetch: Network communication supported by multiple platforms.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "sdk",
```
Comment (Olwiba) [inline package.json]:
```suggestion
  "name": "raygun-sdk",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  "displayName": "sdk",
```
Comment (Olwiba) [inline package.json]:
```suggestion
  "displayName": "raygun-sdk",
```

File: package.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  "version": "1.0.0",
```
Comment (Olwiba) [inline package.json]:
```suggestion
  "version": "0.1.0",
```

Comment (Olwiba) [reply package.json]: As is pre-release?

**Review by Olwiba — APPROVED** [review]
Nice work, rollup.js looks like an interesting tool - should do a knowledge share on this once you're more familiar with it 😄 

There's a bunch of test / foundational stuff but seems find for the init for the project base 🚀

**Review by MattByers — APPROVED** [review]
🥖

**Pull request merged**